specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -302,6 +302,42 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Storage/deletedAccounts": {
```

Comment:
This is a new API, is there any rest spec or feature spec we can reference? 
BTW, for the new API, need ARM review board to review.

Reply:
Shared via email.